cls/_zpkg/isc/sc/git/Socket.cls

```diff
@@ -25,6 +25,20 @@ ClassMethod Run()
 	} ElseIf %request.Get("method") = "pull" {
 		Do ##class(SourceControl.Git.API).Pull()
 	} ElseIf %request.Get("method") = "init" {
+		set root = %request.Get("root")
+
+		// Use user input if provided
+		if (root '= "") && ($extract(root,1,3) '= "e.g") {
```

Comment:
Wait - why do we need to check for e.g? This is the placeholder, not the value; we wouldn't ever send that?